build/config/constants.js

```diff
@@ -25,14 +22,15 @@ const release_config = {
     production: {
         branch        : 'master',
         target_folder : '',
-        valid_sections: ['app', 'app_2', 'all'],
+        valid_sections: ['app_2'],
```

Comment:
do we need `section` anymore?

Reply:
we need too many changes to remove it, we should handle that in https://github.com/binary-com/deriv-app/pull/6 or the release process card.